ts/parts/Pointer.ts

```diff
@@ -1283,15 +1287,21 @@ class Pointer {
         // Normalize before the 'if' for the legacy IE (#7850)
         e = this.normalize(e);
 
-        if ((e as any).button !== 2) {
+        this.onContainerMouseMove(e);
 
-            this.zoomOption(e);
+        if (
+            typeof e.button === 'undefined' ||
+            e.button === 0 // #11635, limiting to primary button
+        ) {
 
-            // issue #295, dragging not always working in Firefox
-            if (e.preventDefault) {
+            this.zoomOption(e);
+            /*
+            // #295, dragging not always working in Firefox
+            // #11635, turn off condition because of wheel click issue
+            if (H.isFirefox) {
                 e.preventDefault();
             }
-
+             */
```

Comment:
Instead of commenting out obsolete code, we should remove it straight away. If a regression happens, we will find the cause using `git blame`.